website/client/src/components/appFooter.vue

```diff
@@ -6,16 +6,12 @@
       <div class="row">
         <div class="col-12 col-md-2">
           <h3>
-            <a
-              href="https://itunes.apple.com/us/app/habitica/id994882113?ls=1&mt=8"
-              target="_blank"
-            >{{ $t('mobileIOS') }}</a>
+            <a href="https://itunes.apple.com/us/app/habitica/id994882113?ls=1&mt=8"
```

Comment:
It looks like you have something like a plugin or a setting in your text editor that seems to change a lot of formatting in files edited, can you disable it?

Reply:
It makes it really difficult to see what actually changed